cmd/build/circleciv2_test.go

```diff
@@ -0,0 +1,49 @@
+package build
+
+import (
+	"strings"
+	"fmt"
+	"testing"
+)
+
+func TestProvider_CircleCIv2(t *testing.T) {
+
+	ctx := mockContext{
+		App:     "my-app",
+		Env:     "dev",
+		Account: "123456789",
+	}
+
+	provider, err := GetProvider("circleciv2")
+	if err != nil {
+		t.Fail()
+	}
+	artifacts, err := provider.ProvideArtifacts(ctx)
+	if err != nil {
+		t.Fail()
+	}
+	if artifacts == nil {
+		t.Fail()
+	}
+	if artifacts[0].FilePath != ".circleci/config.yml" {
+		t.Fail()
+	}
+	t.Log(artifacts[0].FileContents)
+
+	if artifacts[1].FilePath != ".circleci/config.env" {
+		t.Fail()
+	}
+	t.Log(artifacts[1].FileContents)
+
+	configEnv := fmt.Sprintf(`
+export FARGATE_CLUSTER=%v-%v
+export FARGATE_SERVICE=%v-%v
+export REPO=%v.dkr.ecr.us-east-1.amazonaws.com/%v
+export VERSION=0.1.0	
```

Comment:
same thing for these `%v-%v` values; put double-quotes around them just to avoid spurious shell syntax errors.